rooted-ota.sh

```diff
@@ -122,14 +122,18 @@ function createAndReleaseRootedOta() {
 }
 
 function createRootedOta() {
-  [[ "$SKIP_CLEANUP" != 'true' ]] && trap cleanup EXIT ERR
+  [[ "$SKIP_CLEANUP" != 'true' ]] && trap cleanup EXIT ERR || trap cleanupNecessary EXIT ERR
```

Comment:
Why is this necessary? When does the error about requirements happen? And why? Are the permission wrong?

Reply:
Unfortunately, I can't reproduce.
Does this happen every time or only for specific functions or after recovering from errors?

Reply:
> Unfortunately, I can't reproduce.

😕

Let me check this on this weekend!

Reply:
> or only for specific functions or after recovering from errors?

Dunno. 😅

Reply:
> Unfortunately, I can't reproduce.

Ahh, the `--prepatched` stuff is not clear too.

Let me verify my LineageOS port. :confused: